src/shim/fetch.ts

```diff
@@ -1,5 +1,24 @@
-import global from './global';
+import dojoGlobal from './global';
 `!has('build-elide')`;
-import 'whatwg-fetch';
+import 'isomorphic-fetch';
 
-export default global.fetch.bind(global) as (input: RequestInfo, init?: RequestInit) => Promise<Response>;
+import has from '../has/has';
+
+if (typeof global !== 'undefined' && (global as any).fetch && (global as any).fetch !== dojoGlobal.fetch) {
+	dojoGlobal.fetch = (global as any).fetch;
+}
+
+const _fetch = dojoGlobal.fetch.bind(dojoGlobal) as (input: RequestInfo, init?: RequestInit) => Promise<Response>;
+let replacement: (input: RequestInfo, init?: RequestInit) => Promise<Response>;
+
+export default function fetch(input: RequestInfo, init?: RequestInit) {
+	return replacement ? replacement(input, init) : _fetch(input, init);
+}
+
+export function replace(fetch: (input: RequestInfo, init?: RequestInit) => Promise<Response>) {
+	if (has('test')) {
```

Comment:
What is `has` referring to here? Is there a map or a set?

Reply:
https://github.com/dojo/framework/tree/master/src/has/README.md

Reply:
thanks!